python.json

```diff
@@ -94,5 +94,19 @@
   {"id": "py.pydub", "imports": ["pydub"], "name": "Pydub", "repo": "jiaaro/pydub", "tags": ["toolkit"], "tech": ["audio"]},
   {"id": "py.pyechonest", "imports": ["pyechonest"], "name": "Pyechonest", "repo": "echonest/pyechonest", "tags": ["music-search"], "tech": ["api"]},
   {"id": "py.timeside", "imports": ["timeside"], "name": " TimeSide", "repo": "Parisson/TimeSide", "tags": ["web-audio"], "tech": ["audio"]},
-  {"id": "py.tinytag", "imports": ["tinytag"], "name": "tinytag", "repo": "devsnd/tinytag", "tags": ["metadata"], "tech": ["audio"]}
+  {"id": "py.tinytag", "imports": ["tinytag"], "name": "tinytag", "repo": "devsnd/tinytag", "tags": ["metadata"], "tech": ["audio"]},
+
+  {"id": "py.authomatic", "imports": ["authomatic"], "name": "Authomatic", "repo": "authomatic/authomatic", "tags": ["oauth-openid"], "tech": ["python-web"]},
+  {"id": "py.django-allauth", "imports": ["allauth"], "name": "django-allauth", "repo": "pennersr/django-allauth", "tags": ["auth-toolkit"], "tech": ["python-web"]},
+  {"id": "py.django-oauth-toolkit", "imports": ["oauth2_provider"], "name": "Django OAuth Toolkit", "repo": "jazzband/django-oauth-toolkit", "tags": ["oauth"], "tech": ["python-web"]},
+  {"id": "py.flask-oauthlib", "imports": ["flask_oauthlib"], "name": "Flask-OAuthlib", "repo": "lepture/flask-oauthlib", "tags": ["oauth"], "tech": ["python-web"]},
```

Comment:
if those from above go with django, then this one 'flask', otherwise just ignore it :) just not sure, how much you want be specific with the tags

Reply:
I didn't get this.